pandas/core/internals/construction.py

```diff
@@ -91,7 +92,7 @@ def arrays_to_mgr(
     dtype: Optional[DtypeObj] = None,
     verify_integrity: bool = True,
     typ: Optional[str] = None,
-):
+) -> Union[BlockManager, ArrayManager]:
```

Comment:
There is a typing alias for this union (`Manager`, I think)

Reply:
mind keeping this for a follow-up?  id like to get the CI back to green

Reply:
CI checks are also still failing on this PR